third_party/websocket_server/websocket_server.py

```diff
@@ -15,7 +15,6 @@
     from socketserver import ThreadingMixIn, TCPServer, StreamRequestHandler
 
 logger = logging.getLogger(__name__)
-logging.basicConfig()
```

Comment:
Comment it out and add a comment that explains the reasoning in case we end at one point end up updating this dependency, so that we know why it was removed.